js-exercises/update-object/updateObject.test.js

```diff
@@ -0,0 +1,11 @@
+import { updateObject } from "./updateObject";
+
+
+describe("updateObject", () => {
+  test('should tell correct value of obj', () => {
+
+    expect(updateObject(1, '_', ['a', 'b', 'c'])).toEqual([1, "_", Array("a","_","c")]);
```

Comment:
Here in testcase `.toEqual([1, "_", Array("a","_","c")])]` but in the questions expected is `.toEqual(['a', '_', 'c'])`